modules/core/src/main/scala/org/scalasteward/core/git/package.scala

```diff
@@ -21,17 +21,18 @@ import org.scalasteward.core.repoconfig.CommitsConfig
 import org.scalasteward.core.update.show
 
 package object git {
-  def branchFor(update: Update): Branch =
-    Branch(s"update/${update.name}-${update.nextVersion}")
+  def branchFor(update: Update, baseBranch: Branch): Branch =
+    Branch(s"update/${baseBranch.name}/${update.name}-${update.nextVersion}")
 
-  def commitMsgFor(update: Update, commitsConfig: CommitsConfig): String = {
+  def commitMsgFor(update: Update, commitsConfig: CommitsConfig, branch: Branch): String = {
```

Comment:
This is only stylistic, but can we mention the branch only if it is not the default?